enhancements/sig-architecture/68-token-registration/README.md

```diff
@@ -0,0 +1,151 @@
+
+It must be possible for the cluster administrator to specify they wish to use `token` registration in the `clusteradm join` command:
+
+```
+% clusteradm join \
+     --registration=token \
+     --hub-token XXX \
+     --hub-apiserver https://spoke-0.k8s.example.com \
+     --cluster-name spoke-0
+```
+
+#### Story 2 - Spoke administrator joins a cluster to the hub using the default (csr) mechanism
+
+When not specified, we should continue to default to the `csr` registration mechanism. 
+
+```
+% clusteradm join \
+     --hub-token XXX \
+     --hub-apiserver https://spoke-0.k8s.example.com \
+     --cluster-name spoke-0 # default to csr registration
+```
+
+This can also be explicitly set to `csr`, using `--registration=csr` :
+
+```
+% clusteradm join \
+     --registration=csr \
+     --hub-token XXX \
+     --hub-apiserver https://spoke-0.k8s.example.com \
+     --cluster-name spoke-0
+```
+
+#### Story 3 - Hub administrator accepts a spoke cluster using csr or token registration in the same way
+
+From a hub administrator point of viw, the existing `clusteradm accept` command will continue to work, regardless of whether the spoke cluster is using csr or token registration.
+
+```
+% clusteradm accept --clusters spoke-0 # No additional options required if spoke-0 used token registration
+```
+
+#### Story 3 - Spoke Service Account Tokens are refreshed automatically prior to expiry
+
+When the service account token is nearing expiry, the spoke cluster should retrieve a replacement token from the hub, without administrator intervention. 
+
+#### Story 4 - Hub administrator can accept both csr and token registrations on a single hub cluster
+
+In order to future-proof OCM for additional registration types (e.g. cloud provider IAM), it must be possible for a hub to support spoke clusters using both csr and token registration.
+
+#### Story 5 - Hub administrator unjoining a spoke cluster, results in the associated cluster service account being deleted
+
+When a spoke cluster is unjoined, it must no longer be possible to authenticate with the hub using the spoke's Service Account token
+
+#### Story 6 - Procedure for administrators to follow should a token expire or be deleted and the spoke cluster was unable to refresh the token
+
+There needs to be a procedure to follow covering scenarios where the spoke cluster is unable to refresh its service account token, and has lost its ability to authenticate with the hub.
+It should be possible for administrators to restore functionality to the spoke cluster.
+
+Some example scenarios:
+
+1. OCM agents in the spoke cluster were offline (e.g. due to an outage) during which its service account token expired
+2. A network outage resulted in the spoke cluster loosing connectivity to the hub api server for an extended period of time
+3. A service account was intentionally deleted (e.g. the associated token was compromised) and replaced.
+
```

Comment:
we should have another story about addons. I think in token base approach, addons also need to use token to talk to the hub.

Reply:
Added 👍